tests/external_executor/executors/logging_app.py

```diff
@@ -152,7 +152,7 @@ def run_loop(self, activated_event):
                 if "log/private" in request.uri:
                     table = "private:records"
                 elif "log/public" in request.uri:
-                    table = "records"
+                    table = "public:records"
```

Comment:
Good spot, I totally got this wrong! We should also change the private table name from "private:records" to "records", to match the other implementations of this app in JS and C++ (not critical, but eventually we'd like to prove that they do "the same thing", so might as well clean this up now).

Reply:
Done